graphql/validate.lua

```diff
@@ -218,6 +320,17 @@ local visitors = {
     }
   },
 
+  -- <fragmentDefinition>
+  --
+  -- {
+  --   kind = 'fragmentDefinition',
+  --   name = {
+  --     kind = 'name',
+  --     value = <string>,
+  --   },
+  --   typeCondition = <namedType>,
+  --   selectionSet = <selectionSet>,
```

Comment:
I suppose, [optional directives](http://spec.graphql.org/October2016/#FragmentDefinition) are missing

Reply:
It was not there, because the old lpeg based parser does not support it. I looked into [luagraphqlparser](https://github.com/tarantool/luagraphqlparser) and it [appears](https://github.com/tarantool/luagraphqlparser/issues/2) that it does not support it too. We fixed this with @olegrok and it'll be supported soon.

So, I added the directives field here.